ui2.py

```
@@ -0,0 +1,117 @@
+"""
+本代码由[Tkinter布局助手]生成
+官网:https://www.pytk.net
+QQ交流群:905019785
+在线反馈:https://support.qq.com/product/618914
+"""
+import random
```

Ruff (0.11.9)

`random` imported but unused

Remove unused import: `random`

(F401)